packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -32,34 +34,44 @@ module.exports = function compileManualTestHtmlFiles( buildDir, manualTestPatter
 	sourceDirs.forEach( sourceDir => copyStaticFiles( buildDir, sourceDir ) );
 
 	// Generate real HTML files out of the MD + HTML files of each test.
-	sourceMDFiles.forEach( sourceFile => compileTestHtmlFile( buildDir, sourceFile ) );
+	sourceFilePathBases.forEach( sourceFilePathBase => compileTestHtmlFile( buildDir, sourceFilePathBase ) );
+
+	// Watch file and compile on change.
+	watchFiles( sourceMdFiles, ( mdFile ) => compileTestHtmlFile( buildDir, getFilePathWithoutExtension( mdFile ) ) );
+	watchFiles( sourceHtmlFiles, ( mdFile ) => compileTestHtmlFile( buildDir, getFilePathWithoutExtension( mdFile ) ) );
 };
 
-function compileTestHtmlFile( buildDir, sourceFile ) {
+function compileTestHtmlFile( buildDir, sourceFilePathBase ) {
 	const log = logger();
+	const sourceMdFilePath = setExtension( sourceFilePathBase, 'md' );
+	const sourceHtmlFilePath = setExtension( sourceFilePathBase, 'html' );
+	const sourceJsFilePath = setExtension( sourceFilePathBase, 'js' );
+
+	const relativeHtmlFilePath = getRelativeFilePath( sourceHtmlFilePath );
+	const relativeJsFilePath = getRelativeFilePath( sourceJsFilePath );
 
-	const relativeFilePath = getRelativeFilePath( sourceFile );
+	console.log( relativeJsFilePath );
 
-	log.info( `Processing '${ gutil.colors.cyan( sourceFile ) }'...` );
+	log.info( `Processing '${ gutil.colors.cyan( sourceFilePathBase ) }'...` );
 
 	// Compile test instruction (Markdown file).
-	const parsedMarkdownTree = reader.parse( fs.readFileSync( sourceFile, 'utf-8' ) );
+	const parsedMarkdownTree = reader.parse( fs.readFileSync( sourceMdFilePath, 'utf-8' ) );
 	const manualTestInstructions = '<div class="manual-test-sidebar">' + writer.render( parsedMarkdownTree ) + '</div>';
 
 	// Load test view (HTML file).
-	const htmlView = fs.readFileSync( `${ changeExtension( sourceFile, 'html' ) }`, 'utf-8' );
+	const htmlView = fs.readFileSync( `${ sourceHtmlFilePath }`, 'utf-8' );
 
 	// Attach script file to the view.
 	const scriptTag =
 		'<body class="manual-test-container">' +
-			`<script src="./${ changeExtension( path.basename( sourceFile ), 'js' ) }"></script>` +
+			`<script src="/${ relativeJsFilePath }"></script>` +
```

Comment:
Hmm, maybe it's better to name the variable `absoluteToSeverRootJSFilePath`?
The path is relative to the ckeditor5 directory, but it's absolute in context of the server.